### ISSUE TITLE
document render problem in https://dev.px4.io/

### ISSUE THREAD

**Issue by TianmingLi** [PX4/PX4-Devguide#842]
in https://dev.px4.io/v1.9.0/en/airframes/adding_a_new_frame.html

the note render result is like this
```
In short, the output of this mixer would be SERVO = ( (roll input -0.6 + 0) + (pitch input 0.65 + 0) ) * 1 + 0
```

which is confusing

---

the github https://github.com/PX4/Devguide/blob/master/en/airframes/adding_a_new_frame.md

markdown render result is 
```
In short, the output of this mixer would be SERVO = ( (roll input * -0.6 + 0) + (pitch input * 0.65 + 0) ) * 1 + 0
```

correct

**Comment by hamishwillee** [comment]
Thanks! Actually both versions have identical source, and should be rendering badly. This is because the * used to show multiplication is used as markup for "bold". I have created a fix, which is under review. Note that normally I won't fix things older versions - just master.

**Comment by hamishwillee** [comment]
All done (should appear live soon). Note that I did fix v1.9.0 too.
